src/index.js

```diff
@@ -23,7 +23,7 @@ function parse(text, roundUp, momentInstance = moment) {
   let parseString;
 
   if (text.substring(0, 3) === 'now') {
-    time = momentInstance();
+    time = momentInstance(forceNow);
```

Comment:
This would allow consumers to pass in arbitrary strings, which isn't supported in moment, [as pointed out here](https://momentjs.com/docs/#/parsing/string/). 

> Warning: Browser support for parsing strings is inconsistent. Because there is no specification on which formats should be supported, what works in some browsers will not work in other browsers. For consistent results parsing anything other than ISO 8601 strings, you should use String + Format.

Since this doesn't allow the use of `format`, what about adding a check here to ensure that `forceNow` is either a moment instance (`moment.isMoment(forceNow)`) or a native Date object (`forceNow instanceof Date && !isNaN(forceNow.valueOf())`).

It might be nice, to future proof this object for if (hopefully when) we remove `moment` and only allow the value to be a valid Date object.